doc/release-guide.txt

```diff
@@ -47,6 +47,11 @@ procedure should be followed:
 
     git push --follow-tags
 
+* Create a branch for the release (since tags can't be branches)
+
+    git switch -c v${VERSION}-release
+    git push origin v${VERSION}-release
```

Comment:
Is this still (canonically) called `origin`, or should it be abstracted out as (the value of) a shell variable?

(I think in my setup, it's called something else, but that may be mere idiosyncrasy...)